web/packages/extension/assets/options.html

```diff
@@ -57,6 +57,12 @@
                 </select>
                 <label for="preferred_renderer">Preferred renderer</label>
             </div>
+            <div class="option number-input">
+                <input type="number" id="max_execution_duration"
+                       min="1"
+                />
+                <label for="max_execution_duration">How long can actionscript execute for before being forcefully stopped (in seconds)</label>
```

Comment:
nit: actionscript -> ActionScript

Reply:
Whoops, I named it Actionscript instead of ActionScript.